From b53ecec0b5ccb3c0447fc8ef9f8c3fc741e0a897 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?St=C3=A9phane=20Marchesin?= <marcheu@chromium.org>
Date: Fri, 29 Jan 2016 15:31:12 -0800
Subject: [PATCH] modetest: Also print the pixel clock

This can be useful for debugging. xrandr prints it, so why not.
---
 tests/modetest/modetest.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/tests/modetest/modetest.c b/tests/modetest/modetest.c
index 4eb9eac..72cfb6e 100644
--- a/tests/modetest/modetest.c
+++ b/tests/modetest/modetest.c
@@ -247,7 +247,7 @@ static void dump_encoders(struct device *dev)
 
 static void dump_mode(drmModeModeInfo *mode)
 {
-	printf("  %s %d %d %d %d %d %d %d %d %d",
+	printf("  %s %d %d %d %d %d %d %d %d %d %d",
 	       mode->name,
 	       mode->vrefresh,
 	       mode->hdisplay,
@@ -257,7 +257,8 @@ static void dump_mode(drmModeModeInfo *mode)
 	       mode->vdisplay,
 	       mode->vsync_start,
 	       mode->vsync_end,
-	       mode->vtotal);
+	       mode->vtotal,
+	       mode->clock);
 
 	printf(" flags: ");
 	mode_flag_str(mode->flags);
-- 
2.7.0.rc3.207.g0ac5344

